tests/Fixer/Import/OrderedImportsFixerTest.php

```diff
@@ -663,16 +663,36 @@ public function provide70Cases()
     ClassG
 };
 use const some\a\{ConstA, ConstB, ConstC};
+use const some\b\{
+    ConstA,
+    ConstB,
+    ConstC
+};
 use function some\a\{fn_a, fn_b, fn_c};
+use function some\b\{
+    fn_a,
+    fn_b,
+    fn_c
+};
 ',
                 '<?php
 use some\a\{ClassA, ClassB, ClassC as C};
+use function some\b\{
+    fn_b,
+    fn_c,
+    fn_a
+};
 use function some\a\{fn_a, fn_b, fn_c};
 use A\B;
+use const some\b\{
+    ConstC,
+    ConstA,
+    ConstB
+};
 use const some\a\{ConstA, ConstB, ConstC};
 use some\b\{
-    ClassF,
```

Comment:
It is a tweak, so it tests more than before. Nothing is lost, as the `expected` part of this change is also tested. Making a new one would duplicate a lot, so why not just fix this?